org.audacityteam.Audacity.yaml

```diff
@@ -1,10 +1,11 @@
 app-id: org.audacityteam.Audacity
 branch: stable
 runtime: org.freedesktop.Platform
-runtime-version: 1.6
+runtime-version: '1.6'
```

Comment:
Hm, weird, never noticed the missing quotes. Anyway, might as well switch to 18.08.